intro-to-tests/automated-tests.md

```diff
@@ -36,13 +36,22 @@ Each test could follow this generic pattern:
 
 ### Example: Astrology Software
 
+### !callout-info
+
+## Astrological Birth Chart
+An Astrological Birth Chart is a visual map of what the sky looked like when you were born. It is determined using your exact birthdate and birthplace. It includes three signs, your **sun sign** (typically refered to as your Zodiac sign), your **moon sign**, and your **rising sign**. 
+[source](https://www.shape.com/lifestyle/mind-and-body/sun-moon-rising-sign-meaning)
```

Comment:
Birth charts are pretty complicated and lots of people (like me) don't know anything about them.

Reply:
I agree, me too. That's why I added this callout to provide some context. Do you think more context is needed to ensure this example is understandable?